### DISCUSSION TITLE
I am looking for minimumCurrentReleaseAge

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#31810] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github

### Please tell us more about your question or problem

I am looking for a way to configure updates to be pending based on the current version's release age, not just on the new version's release age. The documentation mentions that the alternative to `minimumReleaseAge` is `schedule`, but `schedule` works only if all issues are addressed promptly and there are no accumulated issues.

In our company, we manage a large number of legacy projects with varying levels of commitment to renovate.

The goal is not to replace or neglect `minimumReleaseAge` but to complement it. Here's how I envision it working:

- `minimumCurrentReleaseAge`: Defines when a version becomes too old to be kept.
- `minimumReleaseAge`: Defines when a newly produced version is mature enough to be considered for use.

Example settings:
- `minimumCurrentReleaseAge: 3 years`
- `minimumReleaseAge: 1 year`

This could be interpreted as: update packages that are at least 1 year old but not more than once every 2 years.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
You can try [matchCurrentAge](https://docs.renovatebot.com/configuration-options/#matchcurrentage) along with `minimumReleaseAge` and `schedule` to acheive this.

```json
"packageRules": [
{
 "matchCurrentAge": ">1year",
 "minimumReleaseAge": "2 months",
 "schedule": ["once a year"]   // possibly invalid pls correct before using
}
]
```

**Reply by ns-ggeorgiev** [reply]
thanks, seems exactly what I was looking about

**Comment by HonkingGoose** [comment]
This PR seems a bit related. So you may want to see if that new feature helps you?

- https://github.com/renovatebot/renovate/pull/31729
